src/web/comment/store/commentStore.ts

```diff
@@ -161,6 +161,17 @@ export const deleteComment = (commentId: string) => {
   );
 };
 
+export const hasComments = () => {
+  const comments = useCommentStore.getState().comments;
+  if (comments.length > 0) return true;
+  else return false;
```

Comment:
(nitpick) in most cases like this one, I think conditionally true vs false is just extra verbose without adding readability, compared to just returning the expression that's being evaluated :

```suggestion
  return comments.length > 0;
```

Reply:
"Good point! I've simplified the function as suggested. Thanks for the feedback!" 😊